src/main/java/com/artipie/VertxMain.java

```diff
@@ -106,16 +88,57 @@ public static void main(final String... args) throws IOException, ParseException
         } else {
             throw new IllegalStateException("Storage is not configured");
         }
-        new VertxMain(
-            new Pie(
-                new YamlSettings(
-                    Files.readString(Path.of(storage), Charset.defaultCharset())
-                ),
-                vertx
-            ),
-            vertx,
-            port
-        ).run();
+        final Context ctx = vertx.getOrCreateContext();
+        final JsonObject config = ctx.config();
+        config.put("storage", storage);
+        config.put("port", port);
+        final VertxMain main = new VertxMain();
+        main.init(vertx, ctx);
+        main.start(
```

Comment:
The `start` method is not supposed to be called manually. Read the documentation: https://vertx.io/docs/vertx-core/java/#_verticles

Reply:
You create a verticle, and then deploy it:
```java
Verticle myVerticle = new MyVerticle();
vertx.deployVerticle(myVerticle);
```